src/building/how-to-build-and-run.md

```diff
@@ -168,28 +168,24 @@ there is a (hacky) workaround.  See [the section on "recommended
 workflows"](./suggested.md) below.
 
 Note that this whole command just gives you a subset of the full `rustc`
-build. The **full** `rustc` build (what you get if you say `./x.py build
+build. The **full** `rustc` build (what you get with `./x.py build
 --stage 2 compiler/rustc`) has quite a few more steps:
 
 - Build `rustc` with the stage1 compiler.
   - The resulting compiler here is called the "stage2" compiler.
 - Build `std` with stage2 compiler.
 - Build `librustdoc` and a bunch of other things with the stage2 compiler.
 
-<a name=toolchain></a>
+You almost never need to do this.
 
 ## Build specific components
 
-- Build only the core library
+If you are working on the standard library, you probably don't need to build
+the compiler unless you are planning to use a recently added nightly feature.
+Instead, you can just build stage 0, which uses the current beta compiler.
```

Comment:
I don't think it makes sense to say "build stage 0", it's always either downloaded or pre-installed. "build with stage 0" is what you meant, but I think it's simpler just to leave stages out of it.
```suggestion
Instead, you can just build using the bootstrap compiler.
```

Reply:
Ok.